docs/MigrationGuide.md

```diff
@@ -0,0 +1,58 @@
+# Migration guide from POMI (nri-prometheus) to newrelic-prometheus
+
+What have changed:
+POMI (nri-prometheus) was a custom solution build by New Relic with some of the Prometheus Server capabilities. The `newrelic-prometheus` solution is actually running the Prometheus Server in Agent mode. This allows to use any scrape related feature of the Prometheus Server like authorization methods and relabel configs.
```

Comment:
POMI in the docs is called Prometheus OpenMetric Integration, I think it worth to mention the official name as well